src/components/ThemeToggle.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import { writable } from "svelte/store";
+
+    import { IS_SERVER } from "../scripts/constants";
+
+    const options = ["Light", "Dark"] as const;
+
+    const initialValue = IS_SERVER
+        ? "Dark"
+        : (() => {
+              const stored = localStorage.theme;
+              if (stored && options.includes(stored)) return stored;
+
+              if (window.matchMedia("(prefers-color-scheme: light").matches)
```

Comment:
```suggestion
              if (window.matchMedia("(prefers-color-scheme: light)").matches)
```

but funnily enough this works even without closing it, i love html <3